index.html

```diff
@@ -1,141 +1,182 @@
-  <div class="service-container-2">
+      <article class="service-box-1 box-1"><span class="image-wrapper-2"><img class="icon"
+                                                                              src="./assets/icon_web-development.svg"
+                                                                              alt="icon-web-development"></span><span
+        class="header">Web Development</span>
+        <p>Modern websites need to be responsive and fully functional at all sizes. We develop all of our sites with a
+          mobile first approach and ensure that our code quality is clean, organized and performant. Already have a
+          design but need it developed? We’re here to help!</p></article>
 
-    <article class="service-box-1 box-1">
-      <span class="image-wrapper-2"><img class="icon" src="./assets/icon_web-design.svg" alt="icon-web-design"></span>
-      <span class="header">Logo Design + Branding</span><p>Nike, Apple, Coca-Cola, Starbucks. What do all of these brands have in common? Easily recognizable logos and brand identity. The best brands stand out from the crowd by having a clear and unique point of view. We will work with you to understand your vision and help create a unique brand identity and style guide for your company.</p></article>
 
+      <article class="service-box-1 box-1">
+        <span class="image-wrapper-2"><img class="icon" src="./assets/icon_ux.svg" alt="icon-ux"></span>
+        <span class="header">User Experience</span>
+        <p>In order to build successful products you have to understand the user. We create user personas, wireframes
+          and prototypes to test with users. This allows us to gather feedback which helps to shape the product and can
+          quickly eliminate potential risks – saving you time and money.</p></article>
 
-    <article class="service-box-1 box-1">
-      <span class="image-wrapper-2"><img class="icon" src="./assets/icon_seo.svg" alt=icon-seo></span>
-      <span class="header">SEO</span><p>Do you want your site to rank higher in search results? Google is constantly changing their algorithms for how they rank pages. Don’t worry about constantly playing catch-up. We will make sure your website it optimized, providing you the best opportunities for organic search exposure.</p></article>
+    </div>
+    <div class="service-container-2">
 
+      <article class="service-box-1 box-1">
+        <span class="image-wrapper-2"><img class="icon" src="./assets/icon_web-design.svg" alt="icon-web-design"></span>
+        <span class="header">Logo Design + Branding</span>
+        <p>Nike, Apple, Coca-Cola, Starbucks. What do all of these brands have in common? Easily recognizable logos and
+          brand identity. The best brands stand out from the crowd by having a clear and unique point of view. We will
+          work with you to understand your vision and help create a unique brand identity and style guide for your
+          company.</p></article>
```

Comment:
De tekst en de afbeelding staan door elkaar heen
![Schermafbeelding 2022-10-03 om 19 58 10](https://user-images.githubusercontent.com/23332341/193646661-985febf7-c2ee-4a70-981d-2af79b3d78f3.jpg)

Reply:
Hoi Zwen,

Dat is raar bij is dat namelijk niet zo heb jij hier een verklaring voor ?

Reply:
De afbeelding schaalt niet goed waardoor die door de tekst heen gaat lopen. Als je het venster smaller en breder maakt dan zie je wat er gebeurt